sycl/test-e2e/Regression/commandlist/Inputs/main.cpp

```diff
@@ -172,9 +172,9 @@ int main(int argc, char *argv[]) {
     avgtime += w.run;
 
 #ifdef DEBUG
-    std::chrono::duration<double> d1 = w.start - minstart;
-    std::chrono::duration<double> d2 = w.stop - minstart;
-    std::chrono::duration<double> d3 = w.stop - w.start;
```

Comment:
Same as my other comment. I don't think we need to change this file at all.

Reply:
Removed the type change on host side.
Verified on Linux + Gen12, test can pass with only following change:
```
-            const int upper_bound = sycl::sqrt(1.0 * number) + 1;
+            const int upper_bound = sycl::sqrt(1.0f * number) + 1;
```